lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -6676,367 +6676,326 @@ using namespace Js;
         return newObj;
     }
 
-    struct CompareVarsInfo
+    struct StringItem
```

Comment:
This is new it seems, as well as all string compare functionality.

Reply:
I moved the CompareVarsInfo into the header as I wanted to access it in TypedArray.cpp

StringItem replaces a struct called "Element" previously defined in the header, I went with a different name to be slightly more descriptive and because I re-wrote all the related code.

Reply:
This makes sense, not sure if "StringItem" is an easier name to follow than "Element" :smile: